core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -36,9 +36,18 @@ private TableProperties() {
   public static final String COMMIT_TOTAL_RETRY_TIME_MS = "commit.retry.total-timeout-ms";
   public static final int COMMIT_TOTAL_RETRY_TIME_MS_DEFAULT = 1800000; // 30 minutes
 
-  public static final String COMMIT_NUM_STATUS_CHECKS = "commit.num-status-checks";
+  public static final String COMMIT_NUM_STATUS_CHECKS = "commit.status-checks.num-retries";
```

Comment:
The other properties are in `commit.status-check`, not `commit.status-checks`. Could you remove the extra `s`?

Reply:
Nevermind, I could do it to the PR through github. I'll merge when tests are passing.